requirements-full.txt

```diff
@@ -17,7 +17,7 @@ opencv-python==4.5.1.48
 spikeextractors==0.9.10
 spikeinterface>=0.94.0
 neo==0.10.2
-roiextractors==0.4.17
+roiextractors @ git+https://github.com/catalystneuro/roiextractors.git@155efa5d0597529b5b02a8ada633d8b0452ab1d7
```

Comment:
Can you remind me why this is needed?

Reply:
The dummy segmentation extractor required for testing is a recent addition and the latest commit also added an option for having a dummy segmentation extractor without summary images. As this runs in both our internal and data tests it is required in both requirement files.

Reply:
Ah I see, thanks!